.github/workflows/clean_validation_envs.yml

```diff
@@ -31,4 +31,4 @@ jobs:
           MAIN_TRE_ID: ${{ secrets.TRE_ID }}
           BRANCH_LAST_ACTIVITY_IN_HOURS_FOR_STOP: 2
           BRANCH_LAST_ACTIVITY_IN_HOURS_FOR_DESTROY: 48
-        run: devops/scripts/clean_ci_validation_envs.sh
+        run: .github/scripts/clean_ci_validation_envs.sh
```

Comment:
Is there perhaps a macro for the "root" folder, for more flexibility, to the use of relative path?

Reply:
It's a relative path - from the root of the repo down it the .github holder etc.